typings/index.d.ts

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { ReactTestInstance } from 'react-test-renderer';
+
+export interface GetByAPI {
+  getByName: (name: React.ReactType) => ReactTestInstance;
```

Comment:
`name: React.ReactType | string`, same for `getAllByName`

Reply:
`React.ReactType` already includes string C: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L52

Reply:
Oh, that's cool! Didn't know that :D